lib/ngx/process.md

```diff
@@ -0,0 +1,195 @@
+Name
+====
+
+`ngx.process` - manage the nginx processes for OpenResty/ngx_lua.
+
+Table of Contents
+=================
+
+* [Name](#name)
+* [Status](#status)
+* [Synopsis](#synopsis)
+    * [Enable privilege process and get process type](#enable-privilege-process-and-get-process-type)
+* [Methods](#methods)
+    * [type](#type)
+    * [privilege](#privilege)
+* [Community](#community)
+    * [English Mailing List](#english-mailing-list)
+    * [Chinese Mailing List](#chinese-mailing-list)
+* [Bugs and Patches](#bugs-and-patches)
+* [Author](#author)
+* [Copyright and License](#copyright-and-license)
+* [See Also](#see-also)
+
+Status
+======
+
+This Lua module is currently considered experimental.
```

Comment:
Add a note here saying "The API is still in flux and may change in the future without notice".